protocol/x/revshare/keeper/revshare_test.go

```diff
@@ -353,15 +400,31 @@ func TestKeeper_GetAllRevShares_Valid(t *testing.T) {
 			},
 		},
 		{
-			name:                              "Valid revenue share with no unconditional rev shares",
-			expectedRevShares:                 2,
-			monthlyRollingTakerVolumeQuantums: 1_000_000_000_000, // 1 million USDC
-			expectedTakerFeeRevSharesPpmByAddress: map[string]uint32{
-				constants.BobAccAddress.String(): 150_000, // 15% of taker fees
-			},
-			expectedNetFeeRevSharesPpmByAddress: map[string]uint32{
-				constants.AliceAccAddress.String(): 100_000, // 10% of net fees
+			name: "Valid revenue share with no unconditional rev shares",
+			expectedRevShares: []types.RevShare{
+
+				{
+					Recipient:         constants.BobAccAddress.String(),
+					RevShareFeeSource: types.REV_SHARE_FEE_SOURCE_TAKER_FEE,
+					RevShareType:      types.REV_SHARE_TYPE_AFFILIATE,
+					QuoteQuantums:     big.NewInt(1_500_000),
+				},
+				{
+					Recipient:         constants.AliceAccAddress.String(),
+					RevShareFeeSource: types.REV_SHARE_FEE_SOURCE_NET_FEE,
+					RevShareType:      types.REV_SHARE_TYPE_MARKET_MAPPER,
+					QuoteQuantums:     big.NewInt(1_200_000),
+				},
 			},
+			fill: clobtypes.CreatePerpetualFillForProcess(
+				constants.AliceAccAddress.String(),
+				big.NewInt(10_000_000),
+				constants.BobAccAddress.String(),
+				big.NewInt(2_000_000),
+				big.NewInt(100000),
```

Comment:
Fees are usually in the magnitude of `~0.01%` of fill quote quantums, so let's use more realistic values here